jQuery/ajax.js

```diff
@@ -1,82 +1,113 @@
-function ajax(citizens) {
-  var characters = getUniqueValuesForField('character', citizens);
-  var hometowns = getUniqueValuesForField('hometown', citizens);
-  var sizes = getUniqueValuesForField('size', citizens);
-
-  populateSelectWithOptions("#character-select", characters);
-  populateSelectWithOptions("#hometown-select", hometowns);
-  populateSelectWithOptions("#size-select", sizes);
-
-  var selectHandler = function(event) {
-    var params = gatherParams();
-    var filteredCitizens = filter(params, event.data.citizens);
-    redrawContent(filteredCitizens);
+window.hh = {};
+
+window.hh.selectors = {
+  menu: '.menu',
+  content: '.content',
+  nameInput: '#name-input',
+  characterSelect: '#character-select',
+  hometownSelect: '#hometown-select',
+  sizeSelect: '#size-select'
+};
+
+/* Defines functions that interact directly with the DOM */
+window.hh.domManipulation = window.hh.domManipulation || (function() {
+  function populateSelectWithOptions(select, options) {
```

Comment:
As we continue onwards, if a similar pattern pops up anywhere else I'll do the generalization you've suggested here. I'd still like to have the very specific function there (```populateSelectWithOptions```), just delegating to the more generalized ones, for readability purposes